resources/js/Pages/FAQPage.tsx

```diff
@@ -0,0 +1,51 @@
+
+import React from 'react';
+import { Accordion, AccordionSummary, AccordionDetails, Typography, Box } from '@mui/material';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import Header from '@/Components/Header/header';
+import Footer from '@/Components/Footer/Footer';
+
+
+
+const FAQPage: React.FC = () => {
+  const faqs = [
+    {
+      question: 'What is SportNest?',
+      answer: "SportNest is an online platform that helps users find sports clubs that match their interests, location, and preferences. It also provides useful information, such as required equipment and club details, making it easier for users to start their sports journey."
+    },
+    {
+      question: 'Is SportNest free to use?',
+      answer: 'Yes, users can search for sports clubs and take quizzes for free. Clubs may have additional options or subscription plans depending on the features they need..'
+    },
+    {
+      question: 'How do I find a sports club?',
+      answer: 'You can search for clubs using filters such as location, sport type, age range, and more. You can also take a personalized quiz to get tailored recommendations based on your preferences.'
+    }
+  ];
+
+  return (
```

Comment:
Kun je een linter / formatter runnen hierop

Reply:
linter is da een extensie?

Reply:
Geen idee hoe dat werkt in VSCode. In jetbrains kan ik, right click - format code doen.
<img width="413" alt="image" src="https://github.com/user-attachments/assets/f2256782-b126-45f6-b1ea-db94759a67ca" />

Reply:
ok tis goed nu